packages/form-js-viewer/assets/form-js.css

```diff
@@ -298,19 +398,10 @@
   flex-grow: 1;
 }
 
-.fjs-container .fjs-taglist .fjs-taglist-input:focus-visible {
-  outline: none;
-}
-
-.fjs-container .fjs-taglist .fjs-taglist-dropdown-anchor {
-  position: relative;
-}
-
 .fjs-container .fjs-dropdownlist {
   position: absolute;
   user-select: none;
   overflow-y: auto;
-  scroll-behavior: smooth;
```

Comment:
Why did we remove this?

Reply:
I see: https://github.com/bpmn-io/form-js/pull/377/files#diff-2ff4ceb6cb498575f0c4757cb0df4d062ff331611231231d7d4cb60dc42ce411R91

Reply:
Basically, we're now styling the scroll-behavior in the dropdownlist component. What this allow us to do is to scroll something into view instantly on opening the dropdown, and only then applying the scroll smoothing, which we needed in the case of the time dropdown, as it has to open already scrolled to the value in the field.